config/settings.yml

```diff
@@ -177,6 +177,18 @@ discourse_ai:
   ai_embeddings_semantic_search_enabled:
     default: false
     client: true
+  ai_embeddings_semantic_search_hyde_model:
+    default: "gpt-3.5-turbo"
```

Comment:
I remember this being neccessary for things to work, but I don't know if it's still true after all the changes. Will take a look.